recipes/napari-live-recording/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "napari-live-recording" %}
+{% set version = "0.1.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari_live_recording-{{ version }}.tar.gz
+  sha256: 04c367550953ffe60949e53c0c02d436660aa523047b9c92d833b78cf096ae53
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6,<3.10
+    - setuptools-scm
+  run:
```

Comment:
This package could also use the [ximea APIs](https://www.ximea.com/support/wiki/apis/APIs) but these are not on conda-forge (or PyPI?). If you would be interested, let us know @jacopoabramo. This won't be part of the migration but I can help if you want ;)

Reply:
Hi @jaimergp , yes unfortunately the Ximea APIs for python can only be installed locally by following the indications mentioned by their webpage at this time. I would be happy to load them either into conda-forge or PyPI, I didn't do it myself mostly because I wasn't sure if this would be OK with the company.